examples/documentation_examples/examples_test.go

```diff
@@ -37,18 +37,19 @@ func TestDocumentationExamples(t *testing.T) {
 
 	db := client.Database("documentation_examples")
 
-	documentation_examples.InsertExamples(t, db)
-	documentation_examples.QueryToplevelFieldsExamples(t, db)
-	documentation_examples.QueryEmbeddedDocumentsExamples(t, db)
-	documentation_examples.QueryArraysExamples(t, db)
-	documentation_examples.QueryArrayEmbeddedDocumentsExamples(t, db)
-	documentation_examples.QueryNullMissingFieldsExamples(t, db)
-	documentation_examples.ProjectionExamples(t, db)
-	documentation_examples.UpdateExamples(t, db)
-	documentation_examples.DeleteExamples(t, db)
-	documentation_examples.RunCommandExamples(t, db)
-	documentation_examples.IndexExamples(t, db)
-	documentation_examples.StableAPIExamples()
+	stw := newSubtestWrapper(t, db)
```

Comment:
While I do enjoy the abstraction here, is the usage of `subtestWrapper` strictly necessary? What would you think about just making each of these an explicit subtest? As in:
```
t.Run("InsertExamples", func(t *testing.T) {
   documentation_examples.InsertExamples(t, db)
}
t.Run("QueryToplevelFieldsExamples", func(t *testing.T) {
   documentation_examples.QueryToplevelFieldsExamples(t, db)
}
...
```
That might be more readable?

Reply:
The only real value of the wrapper is to avoid multi-line subtests but that's just cosmetic.  I'm more than happy to do it this way.